javascript/javascript2/week1/homework.js

```diff
@@ -18,3 +20,39 @@ function findShortestWord(words) {
 }
 
 findShortestWord(danishWords); // returns 'ø'
+
+//EXERCISE 2
+
+const danishString = "Jeg har en blå bil";
+const danishString2 = "Blå grød med røde bær";
+const danishCharacters = ["æ", "ø", "å"];
+
+function findDanishLetters(string) {
+  const allLettersArray = string.split("");
+  let count1 = 0;
+  let count2 = 0;
+  let count3 = 0;
+  let total = 0;
+  for (let i = 0; i < allLettersArray.length; i++) {
+    for (let j = 0; j < danishCharacters.length; j++) {
+      if (allLettersArray[i] === danishCharacters[j]) {
```

Comment:
The idea of the two for loops is good, but it is not an also a bit heavy for this. You can try and find some of the implemented JavaScript for string search and also you can try and find other ways of doing this! but overall it works, so good job!

Reply:
good job!